packages/flutter/test/material/chip_test.dart

```diff
@@ -188,6 +188,90 @@ void _expectCheckmarkColor(Finder finder, Color color) {
   );
 }
 
+Future<void> pumpFrames({
+  WidgetTester tester,
+  int frames,
+  Duration duration,
+}) async {
+  for (int i = 0; i < frames; ++i) {
+    await tester.pump(duration);
+  }
+}
+
+Widget _chipWithOptionalDeleteButton({
+  UniqueKey deleteButtonKey,
+  UniqueKey labelKey,
+  bool deletable,
+  TextDirection textDirection = TextDirection.ltr,
+}){
+  return _wrapForChip(
+    textDirection: textDirection,
+    child: Wrap(
+      children: <Widget>[
+        RawChip(
+          onPressed: () {},
+          onDeleted: deletable ? () {} : null,
+          deleteIcon: Icon(Icons.close, key: deleteButtonKey),
+          label: Text(
+            deletable
+              ? 'Chip with Delete Button'
+              : 'Chip without Delete Button',
+            key: labelKey,
+          ),
+        ),
+      ],
+    ),
+  );
+}
+
+bool offsetsAreClose(Offset a, Offset b) => (a - b).distance < 1.0;
+bool radiiAreClose(double a, double b) => (a - b).abs() < 1.0;
+
+// Ripple pattern matches if there exists at least one ripple
+// With the [expectedCenter] and [expectedRadius].
+// This ensures the existence of a ripple.
+PaintPattern ripplePattern(Offset expectedCenter, double expectedRadius) {
+  return paints
+    ..something((Symbol method, List<dynamic> arguments) {
+      if (method != #drawCircle)
+        return false;
+      final Offset center = arguments[0];
+      final double radius = arguments[1];
+      if (offsetsAreClose(center, expectedCenter) && radiiAreClose(radius, expectedRadius))
+        return true;
+      else
+        return false;
+    }
+    );
+}
+
+// Unique ripple pattern matches if there does not exist ripples
+// Other than ones with the [expectedCenter] and [expectedRadius].
```

Comment:
```suggestion
// other than ones with the [expectedCenter] and [expectedRadius].
```
Lowercase because this isn't a new sentence.

Reply:
Fixed.